src/__tests__/imagings/requests/NewImagingRequest.test.tsx

```diff
@@ -1,214 +1,148 @@
-      expect(statusTypesSelect.prop('options')[1].value).toEqual('completed')
-      expect(statusTypesSelect.prop('options')[2].label).toEqual('imagings.status.canceled')
-      expect(statusTypesSelect.prop('options')[2].value).toEqual('canceled')
+      selectEvent.openMenu(dropdownVisits)
+      expect(dropdownVisits).toHaveDisplayValue([''])
+      expect(dropdownVisits.getAttribute('aria-expanded')).toBe('true')
     })
 
-    it('should render a notes text field', async () => {
-      const wrapper = await setup()
-      const notesTextField = wrapper.find(TextFieldWithLabelFormGroup)
+    it('Render a image type input box', async () => {
+      setup()
+      const imgTypeInput = screen.getByPlaceholderText(/imagings\.imaging\.type/i)
+      expect(screen.getByLabelText(/imagings\.imaging\.type/i)).toBeInTheDocument()
 
-      expect(notesTextField).toBeDefined()
-      expect(notesTextField.prop('label')).toEqual('imagings.imaging.notes')
-      expect(notesTextField.prop('isRequired')).toBeFalsy()
-      expect(notesTextField.prop('isEditable')).toBeTruthy()
+      userEvent.type(imgTypeInput, 'tricorder imaging')
+      expect(imgTypeInput).toHaveDisplayValue('tricorder imaging')
     })
 
-    it('should render a save button', async () => {
-      const wrapper = await setup()
-      const saveButton = wrapper.find(Button).at(0)
-      expect(saveButton).toBeDefined()
-      expect(saveButton.text().trim()).toEqual('imagings.requests.create')
+    it('Render a status types select', async () => {
+      setup()
+      const dropdownStatusTypes = screen.getAllByPlaceholderText('-- Choose --')[1]
+      expect(screen.getByText(/patient\.visits\.label/i)).toBeInTheDocument()
+
+      expect(dropdownStatusTypes.getAttribute('aria-expanded')).toBe('false')
+      selectEvent.openMenu(dropdownStatusTypes)
+      expect(dropdownStatusTypes.getAttribute('aria-expanded')).toBe('true')
+      expect(dropdownStatusTypes).toHaveDisplayValue(['imagings.status.requested'])
+
+      const optionsContent = screen
+        .getAllByRole('option')
+        .map((option) => option.lastElementChild?.innerHTML)
+      expect(
+        optionsContent.includes(
+          'imagings.status.requested' && 'imagings.status.completed' && 'imagings.status.canceled',
+        ),
+      ).toBe(true)
     })
 
-    it('should render a cancel button', async () => {
-      const wrapper = await setup()
-      const cancelButton = wrapper.find(Button).at(1)
-      expect(cancelButton).toBeDefined()
-      expect(cancelButton.text().trim()).toEqual('actions.cancel')
+    it('Render a notes text field', async () => {
+      setup()
+      const notesInputField = screen.getByRole('textbox', {
+        name: /imagings\.imaging\.notes/i,
+      })
+      expect(screen.getByLabelText(/imagings\.imaging\.notes/i)).toBeInTheDocument()
+      expect(notesInputField).toBeInTheDocument()
+      userEvent.type(notesInputField, 'Spot likes nutritional formula 221')
```

Comment:
This probably can use an assertion, depending on the test. I will take another look at it. I know I wrote it because of the Star Trek reference lmao
@nobrayner